src/System.Windows.Forms.Primitives/src/Windows/Win32/UI/HiDpi/DPI_AWARENESS_CONTEXT.cs

```diff
@@ -14,18 +14,8 @@ internal readonly partial struct DPI_AWARENESS_CONTEXT
     ///  <see langword="true"/> if the specified context is equal; otherwise, <see langword="false"/> if not equal
     ///  or the underlying OS does not support comparing context.
     /// </returns>
-    public bool IsEquivalent(DPI_AWARENESS_CONTEXT dpiContext)
-    {
-        if (this == UNSPECIFIED_DPI_AWARENESS_CONTEXT && dpiContext == UNSPECIFIED_DPI_AWARENESS_CONTEXT)
-        {
-            return true;
-        }
-
-        if (OsVersion.IsWindows10_1607OrGreater())
-        {
-            return PInvoke.AreDpiAwarenessContextsEqual(this, dpiContext);
-        }
-
-        return false;
-    }
+    public bool IsEquivalent(DPI_AWARENESS_CONTEXT dpiContext) =>
+        (this == UNSPECIFIED_DPI_AWARENESS_CONTEXT && dpiContext == UNSPECIFIED_DPI_AWARENESS_CONTEXT)
+            || (OsVersion.IsWindows10_1607OrGreater()
+            && (bool)PInvoke.AreDpiAwarenessContextsEqual(this, dpiContext));
```

Comment:
nit for readability:
```suggestion
            || (OsVersion.IsWindows10_1607OrGreater()
                && (bool)PInvoke.AreDpiAwarenessContextsEqual(this, dpiContext));
```